README.md

```diff
@@ -31,25 +55,27 @@ and on the public-page alongside a fully functionally WordPress plugin.
 ## Requirements
 
 * Install [Node 8.11.1 or greater](https://nodejs.org)
-* Install [Yarn](https://yarnpkg.com/en/docs/install) (Or npm if you prefer)
+* Install [Yarn](https://yarnpkg.com/en/docs/install) (Or
```

Comment:
shouldn't we also specify the minimum Yarn version, or is this irrelevant? for npm it is implicitly clear, since it's included in node.js (at least on mac os)

Reply:
@lumannnn I added the min. yarn version, good catch.